presto-native-execution/src/test/java/com/facebook/presto/nativeworker/PrestoNativeQueryRunnerUtils.java

```diff
@@ -117,7 +118,9 @@ public static QueryRunner createJavaQueryRunner(Optional<Path> baseDataDirectory
                         ImmutableMap.of(
                                 "parse-decimal-literals-as-double", "true",
                                 "regex-library", "RE2J",
-                                "offset-clause-enabled", "true"),
+                                "offset-clause-enabled", "true",
+                                "internal-communication.jwt.enabled", "true",
```

Comment:
Do we need to enable this by default ?

Reply:
No, my original thought was this would test the Java worker as well. I'll remove it.